sn_faucet/src/faucet_server.rs

```diff
@@ -345,3 +363,125 @@ fn save_address_pk(address: &str, pk_hex: &str) -> Result<()> {
     std::fs::write(addr_path, pk_hex)?;
     Ok(())
 }
+
+async fn distribute_from_maid_to_tokens(
+    client: &Client,
+    snapshot: Snapshot,
+    pubkeys: HashMap<MaidAddress, MaidPubkey>,
+) {
+    for (addr, amount) in snapshot {
```

Comment:
Here I think we can push each iteration into its own `task` and then use `futures::join_all` to collect and ensure that's happening.

And then if this whole function is `tokio::spawn`ed we'll get a fast startup 👍

Reply:
It seemed simpler to just put the whole distribute function in a thread rather than do a bunch of spawning, because transfers can't happen in parallel (wait for wallet lock to be released).